infrastructure/docker/services/postgres/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM postgres:11
```

Comment:
Why ship another Dockerfile if we can juste use the image directly?

Reply:
To standardized everything (docker image name will have the app prefix)
And it allows to customize pg easily

Reply:
Update: https://github.com/jolicode/docker-starter/pull/228/commits/d39dfcc0a4c63692dc1f46777d1303f114b62020